protocols/kad/src/query.rs

```diff
@@ -92,22 +92,33 @@ impl<TInner> QueryPool<TInner> {
         I: IntoIterator<Item = Key<PeerId>>
     {
         let peers = peers.into_iter().map(|k| k.into_preimage()).collect::<Vec<_>>();
-        let parallelism = self.config.replication_factor.get();
+        // TODO: Which parallelism should this use?
+        let parallelism = self.config.replication_factor;
```

Comment:
As of today `ClosestPeersIter` use `ALPHA_VALUE` whereas `FixedPeersIter` uses `K_VALUE`. Why favor the fixed iter over the closest iter @romanb?

I am a bit reluctant exposing two *parallelisms* one for the fixed and one for the closest iter in the `KademliaConfig`. I would prefer making only the latter configurable, adding a comment in the configuration function. What do you think?

Reply:
>  I would prefer making only the latter configurable, adding a comment in the configuration function. What do you think?

Yes, of course. The parallelism for the `FixedPeersIter` needn't be configurable, indeed it may possibly be removed altogether, I only added it for symmetry and wasn't sure if we want to constraint it. In practice, it should currently always be used with a "parallelism" that is essentially equal to the number of peers in the list, i.e. it is always a fan-out.

Reply:
Addressed in https://github.com/libp2p/rust-libp2p/pull/1473/commits/c63922dd0c2f36ba20da7fdeb33514258361464b.